morse_code_decoder.rb

```diff
@@ -0,0 +1,42 @@
+class MorseCodeDecoder
+  MORSE_CODE = {
+    '.-' => 'A',
+    '-...' => 'B',
+    '-.-.' => 'C',
+    '-..' => 'D',
+    '.' => 'E',
+    '..-.' => 'F',
+    '--.' => 'G',
+    '....' => 'H',
+    '..' => 'I',
+    '.---' => 'J',
+    '-.-' => 'K',
+    '.-..' => 'L',
+    '--' => 'M',
+    '-.' => 'N',
+    '---' => 'O',
+    '.--.' => 'P',
+    '--.-' => 'Q',
+    '.-.' => 'R',
+    '...' => 'S',
+    '-' => 'T',
+    '..-' => 'U',
+    '...-' => 'V',
+    '.--' => 'W',
+    '-..-' => 'X',
+    '-.--' => 'Y',
+    '--..' => 'Z'
+  }.freeze
+
+  def self.decode_char(code)
+    MORSE_CODE[code]
+  end
```

Comment:
- [ ] It would be better if the method works and return the expected outcome. Here is a screenshot below to help us understand the problem
![Screenshot from 2023-04-05 23-30-51](https://user-images.githubusercontent.com/46388113/230226236-1959ad73-a8f1-4dc8-9b39-8d2c61c2fa67.png)
![Screenshot from 2023-04-05 23-31-16](https://user-images.githubusercontent.com/46388113/230226249-f63ab03e-8a2f-41bf-b35a-538ee7921a7d.png)

Reply:
Hi @addod19, thanks for the suggestions. As you can see, I encapsulated the methods inside a class. Try running the methods using the class. Here is how it works:

![Untitled](https://user-images.githubusercontent.com/77408085/230237071-7ca1bf39-a1af-4be4-b8bc-1b15b841a7a1.png)

Let me know what you think.